README.md

```diff
@@ -443,4 +443,30 @@ A successful response should create the Work order in DRS and get the list of av
   ]
 }
 ```
+## Get account and address for housing residents
 
+Returns a list of account and address information
+
+```
+Get /v1/CRMAccounts?parisReference=221234470&postcode=E8 1HH
+```
+
+### Parameters
+- Paris reference (required)
+- Postcode (required)
+
+### Response
+A successful response should get a list of account and address information corresponding to the required parameters.
+
+```json
+
+ {
+ "results": [
+		  {
```

Comment:
Are you returning the account number as well? If yes, please add to the response.

Reply:
No, we are not returning the account number. The response I have added is the actual response that the API returns.